flepimop/gempyor_pkg/tests/parameters/test_parameters_class.py

```diff
@@ -0,0 +1,508 @@
+from datetime import date
+from functools import partial
+
+import numpy as np
+import pandas as pd
+import pytest
+from tempfile import NamedTemporaryFile
+
+from gempyor.parameters import Parameters
+from gempyor.testing import create_confuse_subview_from_dict, partials_are_similar
+
+
+class TestParameters:
+    def test_nonunique_parameter_names_value_error(self) -> None:
+        duplicated_parameters = create_confuse_subview_from_dict(
+            "parameters",
+            {"sigma": {"value": 0.1}, "gamma": {"value": 0.2}, "GAMMA": {"value": 0.3}},
+        )
+        with pytest.raises(
+            ValueError,
+            match=(
+                r"Parameters of the SEIR model have the same name "
```

Comment:
do we have an improve-error-message issue around? this message should be indicating the name conflict, and the test here should confirm that it mentions the offending names, rather than trying to match the exact error wording.

Reply:
We do now GH-281.